src/renderer/Widget/AppSpaceListWidget/style.css

```diff
@@ -6,6 +6,8 @@
 }
 .app-space-widget {
   width: 100%;
+  padding-left: 10px;
+  background: white;
```

Comment:
Unnecessary style of `app-space-widget`. Should set horizontal padding for CSS class `widget-header-preview` and `widget-content-preview`.